mmcv/engine/test.py

```diff
@@ -64,21 +64,20 @@ def multi_gpu_test(model, data_loader, tmpdir=None, gpu_collect=False):
     results = []
     dataset = data_loader.dataset
     rank, world_size = get_dist_info()
-    if rank == 0:
-        prog_bar = mmcv.ProgressBar(len(dataset))
     time.sleep(2)  # This line can prevent deadlock problem in some cases.
     for i, data in enumerate(data_loader):
         with torch.no_grad():
             result = model(return_loss=False, **data)
         results.extend(result)
-
-        if rank == 0:
-            batch_size = len(result)
-            batch_size_all = batch_size * world_size
-            if batch_size_all + prog_bar.completed > len(dataset):
-                batch_size_all = len(dataset) - prog_bar.completed
-            for _ in range(batch_size_all):
-                prog_bar.update()
+    if rank == 0:
```

Comment:
Originally the there is only one progressbar, in the new code, it seems that progressbar is created by multiple times?

Reply:
I think the new and old will create progressbar the same number of times.

Reply:
In fact, the new code is not right.
Old code updates the progress bar after one iteration. But new code updates it at the end of iterating the whole dataset which is not an expected behavior.

Reply:
I will fix it.